src/include/optimizer/index_util.h

```diff
@@ -32,6 +32,7 @@ class IndexUtil {
    * This function does not determine whether an index CAN or CANNOT
    * be used. This function only verifies that the preconditions
    * are met before actually searching for a usable index.
+   * TODO(dpatra): once we add support for descending columns in the index change this
```

Comment:
Probably add an assert in this function for now, which yells at you if the sort order is descending

Reply:
Addressed below.